static/scss/detail/tiles.scss

```diff
@@ -97,3 +96,27 @@
     }
   }
 }
+
+.popover {
+  font-family: Rajdhani, "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif !important;
+  font-weight: 600;
+  max-width: 100% !important;
+
+  .popover-header {
+    background-color: white;
+    color: $primary;
+    font-size: larger;
+    text-align: center;
+    border-bottom: 1px solid $primary;
+  }
+
+  .popover-body {
+    text-align: center;
+
+    .date-link {
+      font-weight: 600;
+      font-size: large;
+      color: $blue;
+    }
+  }
+}
```

Comment:
Nitpick: Please add a blank line at the end of this file

Reply:
Done.